src/pages/HomePage/HomePage.css

```diff
@@ -0,0 +1,163 @@
+/* Variable Css*/
+:root {
+  --header-height: 3rem;
```

Comment:
I saw, you are defining `:root` in most of your CSS files, and most of the properties are also same.
So, what you can do, define it at only place inside `App.css`. And still you can use these variables anywhere in your app.